### PR TITLE
complement.sh: Permit skipping docker build

### PR DESCRIPTION
**Pull request by richvdh**
Add a `-f` argument to `complement.sh` to skip the docker build

The docker bits take 10 seconds or so even if everything is cached, so if you just want to rerun some complement tests, it's nice to be able to skip them.

### CODE REVIEW

File: scripts-dev/complement.sh [hunk 1]
```diff
@@ -32,6 +35,24 @@ echo_if_github() {
   fi
 }
 
+# parse our arguments
+skip_docker_build=""
+while [ $# -ge 1 ]; do
+    arg=$1
+    case "$arg" in
+        "-h")
+            echo >&2 "Usage: $0 [-f] <go test arguments>"
```
Comment (DMRobertson) [inline scripts-dev/complement.sh:44]: Might be worth adding a line which tells people to consult `go help testflag` here? (I usually do `go help test` which is less helpful.)

**Review by DMRobertson — APPROVED** [review]
I never knew I wanted this!

File: scripts-dev/complement.sh (hunk 1 above)
Comment (richvdh) [reply scripts-dev/complement.sh]: have added more words.

**Comment by DMRobertson** [comment]
Sytest probably a flake? https://github.com/matrix-org/sytest/issues/1201

File: scripts-dev/complement.sh (hunk 1 above)
Comment (DMRobertson) [reply scripts-dev/complement.sh]: many thanks!

**Pull request merged**